### ISSUE TITLE
Update Sphinx breathe source files

### ISSUE THREAD

**Issue by schuhschuh** [cmake-basis/BASIS#462]
<a href="https://github.com/ahundt"><img src="https://avatars.githubusercontent.com/u/55744?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ahundt](https://github.com/ahundt)**
_Friday Sep 11, 2015 at 21:26 GMT_
_Originally opened as https://github.com/cmake-basis/legacy/issues/462_

---

Breathe has had a lot of development since we last tried it. Perhaps we should try updating the version included and integrating it again! Probably wouldn't be too many code changes since we put it in there once already.

https://breathe.readthedocs.org/en/latest/

https://github.com/michaeljones/breathe

**Comment by ahundt** [comment]
it would be very cool to get rid of the iframe in docs if breathe is up to snuff now

**Comment by schuhschuh** [comment]
But does breathe also support an auto-generation of a complete list of all classes and such? As I understand, including the Doxygen API documentation requires one to write a `.rst` file with an entry for each and every class / function that should be included in the Sphinx output.

**Comment by schuhschuh** [comment]
Seems [doxygenindex](https://breathe.readthedocs.io/en/latest/directives.html#doxygenindex) is the directive I was looking for.

**Comment by ahundt** [comment]
yeah that's the one, there is also [autodoxygenindex](https://breathe.readthedocs.io/en/latest/directives.html#id14)
